Binary files a~/.Makefile.swo and a/.Makefile.swo differ
Binary files a~/.Makefile.swp and a/.Makefile.swp differ
diff -wpruN '--exclude=*.orig' a~/Makefile a/Makefile
--- a~/Makefile	2013-05-14 09:02:00.000000000 +0000
+++ a/Makefile	2023-05-07 18:24:05.491002600 +0000
@@ -29,7 +29,6 @@ cd-discid: $(OBJS)
 
 install: cd-discid
 	$(INSTALL) -D cd-discid $(DESTDIR)$(BINDIR)/cd-discid
-	$(STRIP) $(DESTDIR)$(BINDIR)/cd-discid
 	$(INSTALL) -D -m 644 cd-discid.1 $(DESTDIR)$(MANDIR)/cd-discid.1
 
 clean:
